Gems/AudioSystem/Code/Include/Editor/ACETypes.h

```diff
@@ -39,4 +40,7 @@ namespace AudioControls
 
     using FilepathSet = AZStd::set<AZStd::string>;
 
+    using XmlAllocator = AZ::rapidxml::memory_pool<>;
+    inline static XmlAllocator s_xmlAllocator;
```

Comment:
It doesn't make since to have the variable be both static and inline.
static makes the linkage internal to the translation unit, inline make it so that there is only a single symbol linkage for it, but that is negated by the static in this case.
So only inline should be used in this case.